themis/community/credits.md

```diff
@@ -23,6 +23,7 @@ As for 2020, Themis is supported by core crew:
 
 A special thank you goes to the awesome contributors that help us make Themis better:
 
+- [@maxammann](https://github.com/maxammann) for improvements in wasmThemis code, documentation and triggering fire discussions ([#779](https://github.com/cossacklabs/themis/issues/779), [#781](https://github.com/cossacklabs/themis/pull/781), [#783](https://github.com/cossacklabs/themis/pull/783)).
```

Comment:
More common capitalization, Oxford comma, and you probably meant “flame” there but it's not really *flame* – as in people throwing flagrant insults left and right on the internet:
```suggestion
- [@maxammann](https://github.com/maxammann) for improvements in WasmThemis code, documentation, and triggering discussions ([#779](https://github.com/cossacklabs/themis/issues/779), [#781](https://github.com/cossacklabs/themis/pull/781), [#783](https://github.com/cossacklabs/themis/pull/783)).
```

Reply:
Hall of Fire :P